lib/graphql/define/assign_argument.rb

```diff
@@ -2,16 +2,22 @@ module GraphQL
   module Define
     # Turn argument configs into a {GraphQL::Argument}.
     module AssignArgument
-      def self.call(target, name, type = nil, description = nil, default_value: nil, &block)
+      def self.call(target, name, type = nil, description = nil, **rest, &block)
```

Comment:
The alternative here, though I'm not sure it's worth it, is 

```ruby 
NO_DEFAULT = :__graphql_assign_argument_no_default__
def self.call(..., default_value: NO_DEFAULT)
  # ...
    if default_value != NO_DEFAULT 
      # assign the default value 
    end 
end 
```

The advantage is that we get built-in errors for kwargs. But I guess it's a bit unusual, there's probably no "net decrease" in complexity :P